Framework/CurveFitting/src/FuncMinimizers/DTRSMinimizer.cpp

```diff
@@ -343,14 +343,12 @@ void roots_quadratic(double a0, double a1, double a2, double tol, int &nroots,
     auto pprime = two * a2 * root1 + a1;
     if (pprime != zero) {
       root1 = root1 - p / pprime;
-      p = (a2 * root1 + a1) * root1 + a0;
```

Comment:
These no-effect lines look like there were some minor inconsistencies/leftovers/missing bits in the original code. It would be good if the SCD guys have a look at this.